h/templates/layouts/base.html.jinja2

```diff
@@ -29,6 +29,35 @@
     {% endfor -%}
     {% block styles %}{% endblock %}
 
+    <link rel="apple-touch-icon" sizes="57x57"
```

Comment:
Rebased with just the one apple-touch-icon. The Android icon and the three different sizes of favicon are still there.